packages/babel-plugin/src/styled/__tests__/content-property.test.ts

```diff
@@ -0,0 +1,177 @@
+    const code = `
+      import { styled } from '@compiled/react';
+
+      const hello = 'this is a string';
+      const ListItem = styled.div({
+        content: hello,
+      });
+    `;
+
+    const actual = transform(code, { pretty: true });
+    expect(actual).toContain('._1sb21icm{content:"this is a string"}');
+  });
+
+  it('should add quotes to custom content values', () => {
+    const code = `
+      import { styled } from '@compiled/react';
+
+      const ListItem = styled.div({
+        content: 'hello',
+      });
+    `;
+
+    const actual = transform(code, { pretty: true });
+    expect(actual).toContain('._1sb21e8g{content:"hello"}');
+  });
+
+  it('should not add quotes if they exist already', () => {
+    const code = `
+      import { styled } from '@compiled/react';
+
+      const ListItem = styled.div({
+        content: "'hello'",
+      });
+    `;
+
+    const actual = transform(code, { pretty: true });
+    expect(actual).toContain("._1sb25hbz{content:'hello'}");
+  });
+
+  it('should not add quotes if they exist already (variant)', () => {
+    const code = `
+      import { styled } from '@compiled/react';
+
+      const ListItem = styled.div({
+        content: '"hello"',
+      });
+    `;
+
+    const actual = transform(code, { pretty: true });
+    expect(actual).toContain('._1sb21e8g{content:"hello"}');
+  });
+
+  it("should not add quotes to content values that shouldn't accept them", () => {
+    const code = `
+      import { styled } from '@compiled/react';
+
+      const ListItem = styled.div({
+        '._01 &': { content: 'none' },
+        '._02 &': { content: 'url("http://www.example.com/test.png")' },
+        '._03 &': { content: 'linear-gradient(#e66465, #9198e5)' },
+        '._04 &': {
+          content: 'image-set("image1x.png" 1x, "image2x.png" 2x)',
+        },
+        '._05 &': {
+          content:
+            'url("http://www.example.com/test.png") / "This is the alt text"',
+        },
+        '._06 &': { content: '"prefix"' },
+        '._07 &': { content: 'counter(chapter_counter)' },
+        '._08 &': { content: 'counter(chapter_counter, upper-roman)' },
+        '._09 &': { content: 'counters(section_counter, ".")' },
+        '._10 &': {
+          content:
+            'counters(section_counter, ".", decimal-leading-zero)',
+        },
+        '._11 &': { content: 'attr(value string)' },
+        '._12 &': { content: 'open-quote' },
+        '._13 &': { content: 'close-quote' },
+        '._14 &': { content: 'no-open-quote' },
+        '._15 &': { content: 'no-close-quote' },
+        '._16 &': { content: 'open-quote counter(chapter_counter)' },
+        '._17 &': { content: 'inherit' },
+        '._18 &': { content: 'initial' },
+        '._19 &': { content: 'revert' },
+        '._20 &': { content: 'unset' },
+      });
+    `;
+
+    const actual = transform(code, { pretty: true });
+    const expectedStrings = [
+      '._20 ._dqocn7od{content:unset}',
```

Comment:
I was like "dude, these are completely wrong" oh, it's just ordered 🤣

Reply:
ah right yeah they show up in the snapshot in reverse order (20 to 1), not sure why